operator-framework/src/test/java/io/javaoperatorsdk/operator/ControllerUtilsTest.java

```diff
@@ -1,22 +1,22 @@
-package com.github.containersolutions.operator;
+package io.javaoperatorsdk.operator;
 
-import com.github.containersolutions.operator.sample.TestCustomResource;
-import com.github.containersolutions.operator.sample.TestCustomResourceController;
+import io.javaoperatorsdk.operator.sample.TestCustomResource;
+import io.javaoperatorsdk.operator.sample.TestCustomResourceController;
 import io.fabric8.kubernetes.client.CustomResourceDoneable;
+import io.javaoperatorsdk.operator.api.Controller;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-import static com.github.containersolutions.operator.api.Controller.DEFAULT_FINALIZER;
-import static com.github.containersolutions.operator.sample.TestCustomResourceController.CRD_NAME;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class ControllerUtilsTest {
 
     @Test
     public void returnsValuesFromControllerAnnotationFinalizer() {
-        assertEquals(DEFAULT_FINALIZER, ControllerUtils.getDefaultFinalizer(new TestCustomResourceController(null)));
```

Comment:
This files consist undesired changes related to auto refactoring code using IDEA, but it looks good.